docs/components/novabotflow.md

```diff
@@ -8,6 +8,48 @@
 #Nova.BotFlow
 ##Product Notices
 ***
+
+###2022R4-5 :briefcase: Epic
+####New Features
+- *Nova.Botflow*: Add "Video" processing (FB Messenger) [:clipboard: NC2-177] [NC2-177]
+
+	- Added processing of outgoing videos to the code
+	> .\src\node-red-contrib-chatbot\lib\platforms\facebook\facebook.js
```

Comment:
- do not provide path to the file if it is subjected to change
- poor task description: there are links to the official channel documentation which will light out what was missing